### DISCUSSION TITLE
I don't know how to change colors since 1.7.0

### DISCUSSION THREAD

**Discussion by SolsticeSpectrum** [davatorium/rofi#1445] [Theming] [answered]
This is my original config which I borrowed from somebody else https://github.com/DarkReaper231/blacknwhite/blob/master/.config/rofi/config and I am not sure how to port it. I tried to dump the config but there is nothing about colors or at least it doesn't seems like so.  Could someone please help me port the color theme to 1.7.0?

My rofi originally looked like this.
![68747470733a2f2f692e696d6775722e636f6d2f6a514b463772672e706e67](https://user-images.githubusercontent.com/38611351/132343032-4ba25993-9179-40a0-b344-ad54c81d84be.png)

**Comment by DaveDavenport** [comment, marked as answer]
That format has been deprecated for 4 years.
Its not an easy or perfect conversion:
I did describe it here: https://github.com/davatorium/rofi/discussions/1400
